test/tests_multiple_instances.js

```diff
@@ -0,0 +1,126 @@
+var fs = require("fs");
+function exists(value){
+    return (typeof value != "undefined") ? true : false;
+}
+casper.test.begin("Testing example_multiple_instances.html", 51, function(test) {
```

Comment:
Codacy found an issue: ['casper' is not defined.](https://app.codacy.com/manual/ar2rsawseen/countly-sdk-web/pullRequest?prid=6356797)